src/cli/download_tracker.rs

```diff
@@ -220,6 +220,17 @@ impl DownloadTracker {
     }
 }
 
+struct Eta(Duration);
+
+impl fmt::Display for Eta {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self.0 {
+            Duration::ZERO => Ok(()),
+            _ => f.write_fmt(format_args!("{}", self.0.display())),
```

Comment:
Did you mean something like...

```suggestion
            _ => write!(f, " ETA: {}", self.0.display()),
```

Reply:
Yup, that's better!